tests/integration/targets/connection_proxmox_pct_remote/files/pct

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env bash
+# Copyright (c) 2024 Nils Stein (@mietzen) <github.nstein@mailbox.org>
+# Copyright (c) 2024 Ansible Project
```

Comment:
Since you likely wrote it this year: it's 2025 btw ;-)

Reply:
Happy new year :D